src/containers/pages/IRS/JurisdictionsReport/tests/helpers.test.ts

```diff
@@ -0,0 +1,45 @@
+import * as helpers from '../helpers';
+import * as fixtures from './fixtures';
+
+describe('containers/pages/IRS/JurisdictionsReport/helpers', () => {
+  it('should return the correct columns for Namibia', () => {
+    expect(JSON.stringify(helpers.NamibiaColumns)).toEqual(JSON.stringify(fixtures.NamibiaColumns));
+  });
+
+  it('should return the correct columns for Zambia focus areas', () => {
+    expect(JSON.stringify(helpers.ZambiaFocusAreasColumns)).toEqual(
+      JSON.stringify(fixtures.ZambiaFocusAreasColumns)
+    );
```

Comment:
Is there an advantage to using `.toEqual(someFixture)` instead of `.toMatchSnapshot('someSnapshot')`?

Reply:
> Is there an advantage to using `.toEqual(someFixture)` instead of `.toMatchSnapshot('someSnapshot')`?

According to the link (https://kentcdodds.com/blog/effective-snapshot-testing) shared by @moshthepitt , I agree that snapshots have their drawbacks and should be a last resort. You are better of doing explicit assertions.